DAO/MessageDAO.py

```diff
@@ -161,3 +161,19 @@ def insert_reply(self, message, uid, mid, cid, img=None):
         self.conn.commit()
         return reply_id
 
+    def remove_vote(self, mid, uid, upvote):
```

Comment:
Is this like in facebook? I mean like when you like a post but then hit the like button again to remove the like?

Reply:
Yeah